tests/apps/test_sc_show.py

```
@@ -776,3 +776,80 @@ def test_sc_show_nonexistent_file_with_design(monkeypatch):
+@pytest.mark.timeout(90)
+def test_sc_show_does_not_call_reset_job_params(monkeypatch, make_manifests, asic_gcd):
+    '''Test that Project.show() does not call __reset_job_params.'''
+    make_manifests(asic_gcd)
+
+    # Patch __reset_job_params to verify it's not called
+    with patch.object(asic_gcd, '_Project__reset_job_params') as mock_reset:
+        try:
+            asic_gcd.show()
+        except Exception:
+            # show() might fail due to missing files, that's OK
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)